# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/lvm2/hotfix.patch.musl
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- LVM2.2.03.13/tools/lvmdevices.c.vanilla	2021-08-30 17:09:20.394510297 +0200
+++ LVM2.2.03.13/tools/lvmdevices.c	2021-08-30 17:15:45.733469024 +0200
@@ -16,6 +16,7 @@
 #include "lib/cache/lvmcache.h"
 #include "lib/device/device_id.h"
 #include "lib/device/dev-type.h"
+#include <sys/file.h>
 
 static void _search_devs_for_pvids(struct cmd_context *cmd, struct dm_list *search_pvids, struct dm_list *found_devs)
 {
